### ISSUE TITLE
Select entries for site footer

### ISSUE THREAD

**Issue by graphitefriction** [webtide/jetty.website#4]
The footer for the site is global, i.e. it's displayed at the bottom of every page.

What content, such as images, text, icons, etc., do you want displayed in the footer?

For example, do you want (or are required) to use the footer currently on https://eclipse.dev/jetty/?

If not, the footer can be minimalist, such as the footers on these sites:

- https://docs.squiz.net/developers/latest/documentation/index.html
- https://eclipse.dev/che/docs/stable/overview/introduction-to-eclipse-che/
- https://ipt.gbif.org/manual/en/ipt/latest/

Or, it can be more complex, like:

- https://docs.asciidoctor.org/
- https://docs.couchbase.com/home/server.html
- https://docs.pointsharp.com/

In summary:

1.  What content entries (text, images, icons) do you want displayed in the footer?
2.  What entries should be a link? And what is the link for each entry?
3.  Do you want the entries displayed in a single line (or two), like the minimalist footers, or in columns, like in the more complex footers?
  4.  If in a single line (or two), list the content order and if you want a separator between entries, such as a vertical line, dot, or some other symbol, etc.
  5.  If in columns, what content goes in column 1, column 2, etc.
6.  Or, do you have another layout arrangement you want the footer entries in?

**Comment by mojavelinux** [comment]
Thanks for that info. That's plenty to put together a PR.
